### PR TITLE
Fix legacy sepolicy

### PR DESCRIPTION
**Pull request by goshawk22**
Needed for sdm710 and sdm845 devices

### CODE REVIEW

File: generic/vendor/common/seapp_contexts [hunk 1]
```diff
@@ -34,7 +34,7 @@ user=_app seinfo=platform name=com.qualcomm.timeservice domain=vendor_timeservic
 user=_app seinfo=platform name=.qtidataservices domain=vendor_qtidataservices_app type=app_data_file levelFrom=all
 
 #Add new domain for imshelper service
-user=radio seinfo=platform name=.imshelperservice domain=vendor_imshelper_app type=vendor_radio_data_file
+user=radio seinfo=platform name=.imshelperservice domain=vendor_imshelper_app type=app_data_file
```
Comment (hpnightowl) [inline generic/vendor/common/seapp_contexts:37]: Sure authorship is good? like if known then can you add logs ? 

https://github.com/aospa-ginkgo/device_qcom_sepolicy_vndr/commit/72eb7fe5d0b20865c0c2c95030f8a79bc8ef3a11

File: legacy/vendor/common/init-qti-ims-sh.te [hunk 2]
```diff
@@ -36,4 +36,4 @@ allow init-qti-ims-sh vendor_toolbox_exec:file rx_file_perms;
 set_prop(init-qti-ims-sh, qcom_ims_prop)
 
 # for ro.build.product
-get_prop(init-qti-ims-sh, exported2_default_prop)
+get_prop(init-qti-ims-sh, build_prop)
```
Comment (hpnightowl) [inline legacy/vendor/common/init-qti-ims-sh.te:39]: https://github.com/aospa-ginkgo/device_qcom_sepolicy_vndr/commit/89f16ca36f2c8237304225c1386b6a6a69ccdd08

File: legacy/vendor/common/qtidataservices_app.te [hunk 3]
```diff
@@ -32,7 +32,7 @@ add_hwservice(qtidataservices_app, hal_cacert_hwservice)
 hwbinder_use(qtidataservices_app)
 
 get_prop(qtidataservices_app, cnd_vendor_prop)
-set_prop(qtidataservices_app, exported_radio_prop)
+set_prop(qtidataservices_app, telephony_status_prop)
```
Comment (hpnightowl) [inline legacy/vendor/common/qtidataservices_app.te:35]: https://github.com/aospa-ginkgo/device_qcom_sepolicy_vndr/commit/b49400c6f9023055ec5eb39e2e9a17330450c9ce

**Review by hpnightowl — CHANGES_REQUESTED** [review]

File: generic/vendor/common/seapp_contexts (hunk 1 above)
Comment (adamlawson22) [reply generic/vendor/common/seapp_contexts:37]: Oh my bad I must have messed it up - I'll do it again and push

Comment (hpnightowl) [reply generic/vendor/common/seapp_contexts:37]: Cool ! Nice work Thanks :)

**Pull request closed without merging**